dealership.rb

```diff
@@ -1,29 +1,58 @@
+require 'csv'
```

Comment:
You should delete this `dealership.rb` file since you are not referencing it anywhere.